### PR TITLE
fix(ui):stacktrace dark mode styles

### PR DESCRIPTION
**Pull request by robinrendle**
## Before
CleanShot 2023-01-27 at 15 29 28@2x

## After

CleanShot 2023-01-27 at 15 23 11@2x

### CODE REVIEW

File: static/less/group-detail.less [hunk 1]
```diff
@@ -702,7 +701,6 @@ div.traceback > ul {
       > li.active {
         position: relative;
         z-index: 0;
-        color: #fff;
```
Comment (robinrendle) [inline static/less/group-detail.less:705]: We really need to delete this whole file, but tryna keep this project short and sweet.

File: static/app/utils/theme.tsx [hunk 2]
```diff
@@ -902,6 +902,8 @@ export const lightTheme = {
   button: generateButtonTheme(lightColors, lightAliases),
   tag: generateTagTheme(lightColors),
   level: generateLevelTheme(lightColors),
+  stacktraceActiveBackground: lightColors.gray500,
```
Comment (malwilley) [inline static/app/utils/theme.tsx:905]: I wonder if we want this specific of a variable in our theme file 🤔 @vuluongj20 do you have any opinions on this?

Probably not worth blocking this though

**Review by malwilley — APPROVED** [review]

File: static/app/utils/theme.tsx (hunk 2 above)
Comment (robinrendle) [reply static/app/utils/theme.tsx:917]: Yah, I already synced with @vuluongj20 about this already. I think for now it's okay and it's better than having that nasty .less file style the stacktrace.

Comment (vuluongj20) [reply static/app/utils/theme.tsx:917]: Yeah I don't like one-off color definitions like this either, but as Robin said it's the better approach out of the many worse alternatives we could think of.

**Review by vuluongj20 — APPROVED** [review]

File: static/less/group-detail.less (hunk 1 above)
Comment (vuluongj20) [reply static/less/group-detail.less:705]: 2 lines down, 1177 more to go

**Pull request merged**